### ISSUE TITLE
[Accessibility] No alt text/better description of the Graphs in "Activity"

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4969]
## Describe the bug
The graphs displayed on the "Activity" page lack alternative text or descriptions, making it difficult for screen reader users to understand the content of the page.

### To Reproduce
1. Launch the local host http://localhost:4200/
2. Navigate to "Activity" link http://localhost:4200/activity
3. This will open the Activity page, which contains graphs.
4. Use a screen reader and try to navigate the page. Observe whether the graphs are properly described and if you can understand the purpose of the page.

### Expected behavior
All users, including those using screen readers, should be able to clearly understand the purpose of the Activity page and the information conveyed by the graphs.

### Screenshots
[Screencast from 17-06-2024 07:34:33 ASUBUHI.webm](https://github.com/ushahidi/platform-client-mzima/assets/124133577/ce1ba45a-8b59-47c2-967b-1c9572d7b393)

**Comment by Angamanga** [comment]
Closing but keeping in Linear https://linear.app/ushahidi/issue/PLAT-5070/[accessibility]-no-alt-textbetter-description-of-the-graphs-in
